src/constants.ts

```diff
@@ -7,6 +7,7 @@ export default {
 	pmUrl: process.env.PM_URL,
 
 	todoUrl: 'https://github.com/Lauraducky/pockybot/issues',
+	googleUrl: 'https://storage.googleapis.com/',
```

Comment:
So we no longer user the fileURL, so should we delete it and the associated environment variable and teamcity stuff?

Reply:
I think so